src/Terminus.php

```diff
@@ -75,6 +75,8 @@ public function __construct(Config $config, InputInterface $input = null, Output
 
         $this->configureContainer();
 
+        $this->setLogger($container->get('logger'));
```

Comment:
@greg-1-anderson Do you suppose there'll be drawbacks to simply moving this here?

Reply:
Yeah that looks perfect. Seems the Terminus class was logger aware, but never set up its logger.